rmc-docs/src/limitations.md

```diff
@@ -0,0 +1,413 @@
+fmaf64 | Yes | |
+fmul_fast | No | |
+forget | Partial | Generates `SKIP` statement |
+frem_fast | No | |
+fsub_fast | No | |
+likely | Yes | |
+log2f32 | Yes | |
+log2f64 | Yes | |
+log10f32 | Yes | |
+log10f64 | Yes | |
+logf32 | Yes | |
+logf64 | Yes | |
+maxnumf32 | Yes | |
+maxnumf64 | Yes | |
+min_align_of | Yes | |
+min_align_of_val | Yes | |
+minnumf32 | Yes | |
+minnumf64 | Yes | |
+move_val_init | No | |
+mul_with_overflow | Yes | |
+nearbyintf32 | Yes | |
+nearbyintf64 | Yes | |
+needs_drop | Yes | |
+nontemporal_store | No | |
+offset | Partial | Missing undefined behavior checks |
+powf32 | Yes | |
+powf64 | Yes | |
+powif32 | Yes | |
+powif64 | Yes | |
+pref_align_of | Yes | |
+prefetch_read_data | No | |
+prefetch_read_instruction | No | |
+prefetch_write_data | No | |
+prefetch_write_instruction | No | |
+ptr_guaranteed_eq | Partial | |
+ptr_guaranteed_ne | Partial | |
+ptr_offset_from | Partial | Missing undefined behavior checks |
+raw_eq | Partial | Missing undefined behavior checks |
+rintf32 | Yes | |
+rintf64 | Yes | |
+rotate_left | Yes | |
+rotate_right | Yes | |
+roundf32 | Yes | |
+roundf64 | Yes | |
+rustc_peek | No | |
+saturating_add | Yes | |
+saturating_sub | Yes | |
+sinf32 | Yes | |
+sinf64 | Yes | |
+size_of | Yes | |
+size_of_val | Yes | |
+sqrtf32 | Yes | |
+sqrtf64 | Yes | |
+sub_with_overflow | Yes | |
+transmute | Yes | |
+truncf32 | Yes | |
+truncf64 | Yes | |
+try | No | |
+type_id | Yes | |
+type_name | Yes | |
+unaligned_volatile_load | Partial | See [Observations - Concurrency](#concurrency) |
+unaligned_volatile_store | No | See [Observations - Concurrency](#concurrency) |
+unchecked_add | Yes | |
+unchecked_div | Yes | |
+unchecked_mul | Yes | |
+unchecked_rem | Yes | |
+unchecked_shl | Yes | |
+unchecked_shr | Yes | |
+unchecked_sub | Yes | |
+unlikely | Yes | |
+unreachable | Yes | |
+variant_count | No | |
+volatile_copy_memory | Partial | See [Observations - Concurrency](#concurrency) |
+volatile_copy_nonoverlapping_memory | Partial | See [Observations - Concurrency](#concurrency) |
+volatile_load | Partial | See [Observations - Concurrency](#concurrency) |
+volatile_set_memory | No | See [Observations - Concurrency](#concurrency) |
+volatile_store | No | See [Observations - Concurrency](#concurrency) |
+wrapping_add | Yes | |
+wrapping_mul | Yes | |
+wrapping_sub | Yes | |
+write_bytes | Yes | |
+
+#### Atomics
+
+All atomic intrinsics are compiled as an atomic block where the operation is
+performed. But as noted in [Observations - Concurrency](#concurrency), CBMC
+support for concurrent verification is limited and not used by default.
+Verification on code containing atomic intrinsics should not be trusted given
+that CBMC assumes the code to be sequential.
+
+#### SIMD instructions
+
+While RMC is capable of generating code for SIMD instructions. Unfortunately,
+CBMC does not provide support for some operations like vector comparison (e.g.,
+`simd_eq`) or shuffling (e.g., `simd_shuffle8`).
```

Comment:
wait, I thought I added shuffle.

Reply:
Sorry, I thought this was not supported by CBMC but after reviewing related issues I see that you implemented shuffle operations in RMC. So I removed shuffling from here.